src/scripts/main.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const spiderSize = spider.offsetWidth;
+
 document.addEventListener('click', e => {
-  // write code here
+  const wall = e.target.closest('.wall');
+
+  if (!wall) {
+    return;
+  }
+
+  const topGap = wall.offsetTop + wall.clientTop;
+  const leftGap = wall.offsetLeft + wall.clientLeft;
+
+  const positionX = e.clientX - leftGap - spiderSize / 2;
+  const positionY = e.clientY - topGap - spiderSize / 2;
+
```

Comment:
use e.offsetX and e.offsetY instead of e.clientX - leftGap and e.clientY - topGap